build_scripts/docker/clean.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
We could use cleanup scripts as a final step of build scripts by default?

Reply:
The cleanup scripts are mostly a convenience. We don't want to clean up by default, because you may want to inspect some of the intermediate artifacts if the build fails.